README.md

```diff
@@ -246,71 +203,114 @@ vi config.yaml
 ```
 
 ```yaml
-name: "chainHero-network"
-
-# Describe what the target network is/does.
-description: "The network which will host my first blockchain"
-
+name: "heroes-service-network"
+#
 # Schema version of the content. Used by the SDK to apply the corresponding parsing rules.
-version: 2
+#
+version: 1.0.0
 
+#
 # The client section used by GO SDK.
+#
 client:
+
   # Which organization does this application instance belong to? The value must be the name of an org
-  organization: Org1
+  # defined under "organizations"
+  organization: org1
+
   logging:
     level: info
 
   # Global configuration for peer, event service and orderer timeouts
-  peer:
-    timeout:
-      connection: 3s
-      queryResponse: 45s
-      executeTxResponse: 30s
-  eventService:
-    timeout:
-      connection: 3s
-      registrationResponse: 3s
-  orderer:
-    timeout:
-      connection: 3s
-      response: 5s
-
-  # Root of the MSP directories with keys and certs. The Membership Service Providers is component that aims to offer an abstraction of a membership operation architecture.
+  # if this this section is omitted, then default values will be used (same values as below)
+#  peer:
+#    timeout:
+#      connection: 10s
+#      response: 180s
+#      discovery:
+#        # Expiry period for discovery service greylist filter
+#        # The channel client will greylist peers that are found to be offline
+#        # to prevent re-selecting them in subsequent retries.
+#        # This interval will define how long a peer is greylisted
+#        greylistExpiry: 10s
+#  eventService:
+#    # Event service type (optional). If not specified then the type is automatically
+#    # determined from channel capabilities.
+#    type: (deliver|eventhub)
+    # the below timeouts are commented out to use the default values that are found in
+    # "pkg/fab/endpointconfig.go"
+    # the client is free to override the default values by uncommenting and resetting
+    # the values as they see fit in their config file
+#    timeout:
+#      connection: 15s
+#      registrationResponse: 15s
+#  orderer:
+#    timeout:
+#      connection: 15s
+#      response: 15s
+#  global:
+#    timeout:
+#      query: 180s
+#      execute: 180s
+#      resmgmt: 180s
+#    cache:
+#      connectionIdle: 30s
+#      eventServiceIdle: 2m
+#      channelConfig: 30m
+#      channelMembership: 30s
+#      discovery: 10s
+#      selection: 10m
+
+  # Root of the MSP directories with keys and certs.
   cryptoconfig:
-    path: "${GOPATH}/src/github.com/chainHero/heroes-service/fixtures/crypto-config"
+    path: ${GOPATH}/src/github.com/chainHero/heroes-service/fixtures/crypto-config
 
-  # Some SDKs support pluggable KV stores, the properties under "credentialStore" are implementation specific
+  # Some SDKs support pluggable KV stores, the properties under "credentialStore"
+  # are implementation specific
   credentialStore:
-    path: "/tmp/heroes-service-kvs"
+    path: /tmp/state-store
```

Comment:
You nee dot set `heroes-service-store` to keep the clean task of the Makefile valid.